CHANGES

```diff
@@ -0,0 +1,23 @@
+Changes
+=======
+
+Version 0.3.0 (released 2015-03-17)
+
+- New method `has_active_child(recursive=True)` in `MenuEntryMixin`.  (#25)
+- Fixed documentation of blueprint example. (#21)
+- Configuration for Docker and demo app. (#22 #29)
+- Fixed template example and added code block types.  (#14)
+
+Version 0.2.0 (released 2014-11-04)
+
+- The Flask-Menu extension is now released under more permissive
+  Revised BSD License. (#12)
+- New support for additional keyword arguments stored as `MenuItem`
+  attributes. (#19)
+- Richer quick-start usage example. (#16)
+- Support for Python 3.4. (#6)
+- Documentation improvements. (#3)
+
+Version 0.1.0 (released 2014-06-27)
```

Comment:
Cosmetics; `FIXME` is more usual than `TBD`, e.g. besides editor warnings I'm grepping for this from time to time.